TodoApp/SceneDelegate.swift

```diff
@@ -13,7 +13,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
 
-    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    func scene(_ scene: UIScene, willConnectTo mysuperlonglonglonglonglonglonglonglonglongsession: UISceneSession, options mysuperlonglonglonglonglonglonglonglonglonglongconnectionOptions: UIScene.ConnectionOptions) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/coutar-t/Bitrise-CI/blob/542b9b0a748dff3bd6f40f08fe5332fde2be59a7/TodoApp/SceneDelegate.swift#L16"></span>Variable name should be between 3 and 40 characters long: ‘mysuperlonglonglonglonglonglonglonglonglonglongconnectionOptions’<br />
<code>identifier_name</code> <code>SceneDelegate.swift:16</code></td>
    </tr>
  </tbody>
</table>